README.md

```diff
@@ -10,202 +10,76 @@
 
-The system requirements vary depending on platform and are available on dedicated pages:
-- [Linux](https://docs.openvino.ai/2024/get-started/install-openvino/install-openvino-linux.html)
-- [Windows](https://docs.openvino.ai/2024/get-started/install-openvino/install-openvino-windows.html)
-- [macOS](https://docs.openvino.ai/2024/get-started/install-openvino/install-openvino-macos.html)
+## Documentation
 
-## How to build
+[**User documentation**](https://docs.openvino.ai/)
 
-See [How to build OpenVINO](./docs/dev/build.md) to get more information about the OpenVINO build process.
+Contains detailed information about OpenVINO and guides you from installation through optimizing and deploying models for your AI applications.
 
-## How to contribute
+[**Developer documentation**](./docs/dev/index.md)
 
-See [Contributions Welcome](https://github.com/openvinotoolkit/openvino/issues/17502) for good first issues.
+Focuses on how OpenVINO [components](./docs/dev/index.md#openvino-components) work and describes [building](./docs/dev/build.md)  and [contributing](./CONTRIBUTING.md) processes.
 
-See [CONTRIBUTING](./CONTRIBUTING.md) for contribution details. Thank you!
+## Contribution and Support
 
-Visit [Intel DevHub Discord server](https://discord.gg/7pVRxUwdWG) if you need help or wish to talk to OpenVINO developers. You can go to the channel dedicated to Good First Issue support if you are working on a task.
+Check out [Contribution Guidelines](./CONTRIBUTING.md) for more details.
+Read the [Good First Issues section](./CONTRIBUTING.md#3-start-working-on-your-good-first-issue), if you're looking for a place to start contributing. We welcome contributions of all kinds!
 
-## Take the issue
-If you wish to be assigned to an issue please add a comment with `.take` command.
+You can ask questions and get support on:
 
-## Get support
+* [GitHub Issues](https://github.com/openvinotoolkit/openvino/issues).
+* OpenVINO channels on the [Intel DevHub Discord server](https://discord.gg/7pVRxUwdWG).
+* The [`openvino`](https://stackoverflow.com/questions/tagged/openvino) tag on Stack Overflow\*.
 
-Report questions, issues and suggestions, using:
+## Additional Resources
 
-* [GitHub* Issues](https://github.com/openvinotoolkit/openvino/issues)
-* The [`openvino`](https://stackoverflow.com/questions/tagged/openvino) tag on StackOverflow\*
-* [Forum](https://software.intel.com/en-us/forums/computer-vision)
-* OpenVINO channels on the [Intel DevHub Discord server](https://discord.gg/7pVRxUwdWG)
+* [Product Page](https://software.intel.com/content/www/us/en/develop/tools/openvino-toolkit.html)
+* [Release Notes](https://docs.openvino.ai/2024/about-openvino/release-notes-openvino.html)
+* [OpenVINO Blog](https://blog.openvino.ai/)
+* [OpenVINO™ toolkit on Medium](https://medium.com/@openvino)
+* [Telemetry](https://docs.openvino.ai/2024/about-openvino/additional-resources/telemetry.html)
```

Comment:
```suggestion
```
I see this section as additional educational resources about OpenVINO

Reply:
sure, if it's not required (question to @moslex), let's remove this link

Reply:
> sure, if it's not required (question to @moslex), let's remove this link

Ok to remove